opentelemetry/instrumentation/app/work.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"context"
+	"math/rand"
+	"time"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+)
+
+const name = "work"
```

Comment:
Should we make this a full name? `github.com/GoogleCloudPlatform/golang-samples/opentelemetry/instrumentation/app`

Reply:
Is that the convention in Go ? I was following suit from [rolldice example](https://github.com/open-telemetry/opentelemetry-go/blob/main/example/dice/rolldice.go).

Reply:
Yes, it is the convention: https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-go-contrib%20scopeName&type=code

I opened an issue in the otel-go repo as well to update the rolldice example

Reply:
I will make this change then.